### DISCUSSION TITLE
How can I get skin id

### DISCUSSION THREAD

**Discussion by yarikdevcom** [saul/demofile#315] [Questions] [answered]
I tried to find an inside the weapon.props not found anything

**Comment by saul** [comment]
What are you referring to by skin ID exactly? Does `weapon.itemName` give you what you want?

**Reply by yarikdevcom** [reply]
itemName is only AK-47 but I need for example "Neon Rider" etc. color skins that model using, I will pay for upgrading lib, give me a price

**Comment by yarikdevcom** [comment]
How I can get info from CBaseAttributableItem, where I can find it

**Comment by saul** [comment, marked as answer]
I've just opened a PR to make the paint kit available in `player_death` events. See https://github.com/saul/demofile/pull/352.

Unfortunately there's no other way to get this from a demo file. You'd have to use the Steam APIs to get the paint kit for a given item ID. You can get the item ID with:

```ts
const itemId = new Long(
  weapon.getProp("DT_ScriptCreatedItem", "m_iItemIDLow"),
  weapon.getProp("DT_ScriptCreatedItem", "m_iItemIDHigh"),
  true
);
```
